src/main/java/co/rsk/bitcoinj/script/RedeemScriptParser.java

```diff
@@ -9,11 +9,8 @@ public interface RedeemScriptParser {
     enum MultiSigType {
         NO_MULTISIG_TYPE,
```

Comment:
This one should also go away, right?

Reply:
No, because it is being used by the `NonStandardErpRedeemScriptParserHardcoded`. But I think it could be renamed to something more specific to what it is being used for e.g.: `NonStandardTestnetErpHardcoded`, wdyt?